From b8013aa2f0eafe8ec7fdd919b33c5277d26cb925 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Mon, 5 Feb 2018 09:35:01 +0000
Subject: [PATCH 211/277] mmc: bcm2835-sdhost: Add include

highmem.h (needed for kmap_atomic) is pulled in by one of the other
include files, but only with some CONFIG settings. Make the inclusion
explicit to cater for cases where the CONFIG setting is absent.

See: https://github.com/raspberrypi/linux/issues/2366

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 drivers/mmc/host/bcm2835-sdhost.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mmc/host/bcm2835-sdhost.c b/drivers/mmc/host/bcm2835-sdhost.c
index 6c0dafe6c64b..273b1be058c5 100644
--- a/drivers/mmc/host/bcm2835-sdhost.c
+++ b/drivers/mmc/host/bcm2835-sdhost.c
@@ -52,6 +52,7 @@
 #include <linux/time.h>
 #include <linux/workqueue.h>
 #include <linux/interrupt.h>
+#include <linux/highmem.h>
 #include <soc/bcm2835/raspberrypi-firmware.h>
 
 /* For mmc_card_blockaddr */
-- 
2.16.1

